### DISCUSSION TITLE
Every time the repo is mentioned in the python file, it should open in the web browser.

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [ThioJoe/YT-Spammer-Purge#190] [Ideas]
Using something called webbrowser, we can do this: https://pypi.org/project/pycopy-webbrowser/

the code to do this is:
import webbrowser
webbrowser.open('https://github.com/ThioJoe/YouTube-Spammer-Purge')

**Comment by TechStudent10** [comment]
Pretty sure that's gonna be annoying.

**Comment by KendallDoesCoding** [comment]
True. I meant if they tapped the github. But, idk of a way for now.

**Comment by KendallDoesCoding** [comment]
Taken from #191 

> @ThioJoe How does the google authenticate link open in the browser automatically though in the .exe atleast. Means, there is a way to get this done?

> https://github.com/ThioJoe/YT-Spammer-Purge/blob/main/Scripts/auth.py#L74 -> https://github.com/googleapis/google-auth-library-python-oauthlib/blob/main/google_auth_oauthlib/flow.py#L468

@ThioJoe Please check this out.
